src/sentry/static/sentry/app/components/repositoryFileSummary.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import FileChange from './fileChange';
+import {tn} from '../locale';
+
+function Collapsed(props) {
+  return (
+    <li className="list-group-item list-group-item-sm align-center">
+      <span className="icon-container">
+      </span>
+      <a onClick={props.onClick}>{tn(('Show ' + props.count + ' collapsed file'), ('Show ' + props.count + ' collapsed files'), props.count)}</a>
```

Comment:
i think you can just do:
`<a onClick={props.onClick}>{tn(('Show %d collapsed file'), ('Show %d collapsed files'), props.count)}</a>`
can't you?